build_runner/lib/src/asset/reader.dart

```diff
@@ -24,10 +22,10 @@ class SinglePhaseReader implements AssetReader {
   final Set<AssetId> _assetsRead = new Set();
   final AssetGraph _assetGraph;
   final int _phaseNumber;
-  final String _rootPackage;
+  final String _primaryPackage;
```

Comment:
[nit] I'm not sure this name is more clear. `PackageGraph` still calls this `root` right? FWIW I think 'root' is pretty clear since it's also the 'workspace root' when we build.